app/upgrades/v2/upgrades.go

```diff
@@ -5,7 +5,6 @@ import (
 	"github.com/cosmos/cosmos-sdk/types/module"
 	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
 
-	claimstypes "github.com/evmos/evmos/v9/x/claims/types"
```

Comment:
isnt it better to just delete the old upgrades files?

Reply:
sure. How many updates back should we support?

Reply:
Only 3 max, current + last 2 ones for reference (eg: next release is v10, we should support v9 and v8)